document/src/main/java/com/ritense/document/web/rest/impl/JsonSchemaDocumentDefinitionResource.java

```diff
@@ -61,6 +62,22 @@ public ResponseEntity<Page<? extends DocumentDefinition>> getDocumentDefinitions
         return ok(documentDefinitionService.findAll(fixPageable(pageable)));
     }
 
+    @Override
+    public ResponseEntity<String> getDocumentDefinitionTemplate(DocumentDefinitionTemplateRequestDto requestDto) {
+        return ok(
+            """
+                {
+                    "$id": "%s.schema",
```

Comment:
Shouldn't we sanatize the input? I could break this JSON by sending `xyz"`. Best would be to use Jackson to set/add a TextNode I think.

Reply:
Added sanitization